File: 1635-number-of-good-pairs/number-of-good-pairs.cpp
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        int n=nums.size();
        int answer=0;
        for(int i=0;i<n;i++){
            for(int j=i;j<n;j++){
                if(nums[i]==nums[j]&&i<j){
                    answer++;
                }
            }
        }
        return answer;
    }
};

File: 1635-number-of-good-pairs/number-of-good-pairs.go
func numIdenticalPairs(nums []int) int {
    count:=0
    for index1,num1:=range nums{
        for index2,num2:=range nums{
            if num1==num2 && index1<index2{
                count++
            }
        }
    }
    return count
}